integration-libs/opf/base/occ/opf-base-occ.module.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { provideDefaultConfig } from '@spartacus/core';
+import {
+  OpfOrderAdapter,
+  OpfPaymentAdapter,
+  OtpAdapter,
+} from '@spartacus/opf/base/core';
+import { OccOrderNormalizer } from '@spartacus/order/occ';
+import { ORDER_NORMALIZER } from '@spartacus/order/root';
+import { OccOpfOrderAdapter, OccOtpAdapter } from './adapters';
+import { OccOpfPaymentAdapter } from './adapters/occ-opf.adapter';
+import { defaultOccOpfConfig } from './config/default-occ-opf-config';
+import { defaultOccOpfOrderConfig } from './config/default-occ-opf-order-config';
+import { defaultOccOtpConfig } from './config/default-occ-otp-config';
+
+@NgModule({
+  imports: [CommonModule],
+  providers: [
+    provideDefaultConfig(defaultOccOpfConfig),
+    {
+      provide: OpfPaymentAdapter,
+      useClass: OccOpfPaymentAdapter,
+    },
+    provideDefaultConfig(defaultOccOpfOrderConfig),
+    {
+      provide: OpfOrderAdapter,
+      useClass: OccOpfOrderAdapter,
+    },
+    {
+      provide: ORDER_NORMALIZER,
+      useExisting: OccOrderNormalizer,
+      multi: true,
+    },
```

Comment:
[QUESTION]

Do we _really_ need to provide the original `OccOrderNormalizer` here? I can see it's already provided in [unit-order-occ.module.ts#L22-L26](https://github.com/SAP/spartacus/blob/e3b1abd4b857384143b2c8165746d426985d2a9e/feature-libs/organization/unit-order/occ/unit-order-occ.module.ts#L22-L26)